### PR TITLE
Tests/install pkg

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

add extra os for pkg install

## PR review    
Anyone in the community is free to review the PR once the tests have passed.     
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

**Comment by Borda** [comment]
strange error with Windows install 
```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "D:\a\pytorch-lightning\pytorch-lightning\pytorch_lightning\__init__.py", line 56, in <module>
    from pytorch_lightning.core import LightningDataModule, LightningModule
  File "D:\a\pytorch-lightning\pytorch-lightning\pytorch_lightning\core\__init__.py", line 361, in <module>
    from pytorch_lightning.core.datamodule import LightningDataModule
  File "D:\a\pytorch-lightning\pytorch-lightning\pytorch_lightning\core\datamodule.py", line 21, in <module>
    from torch.utils.data import DataLoader
ModuleNotFoundError: No module named 'torch'
```

**Comment by justusschock** [comment]
@Borda this is because there is no pip package for torch on windows. We either have to install from file specifying the link to PyTorch page (see official install instructions) or install without deps

**Comment by Borda** [comment]
> @Borda this is because there is no pip package for torch on windows. We either have to install from file specifying the link to PyTorch page (see official install instructions) or install without deps

so adding a path to packages shall fix it, right?
`--find-links https://download.pytorch.org/whl/cpu/torch_stable.html`

File: .github/workflows/install-pkg.yml [hunk 1]
```diff
@@ -39,25 +20,33 @@ jobs:
...
       - name: Install package
         run: |
-          pip install virtualenv
-          virtualenv vEnv ; source vEnv/bin/activate
+          # pip install virtualenv
+          # virtualenv vEnv --system-site-packages
+          # source vEnv/bin/activate
           # pip install -r requirements.txt
-          pip install dist/*
+          pip install --find-links https://download.pytorch.org/whl/torch_stable.html dist/*
```
Comment (justusschock) [inline .github/workflows/install-pkg.yml:49]: Have you tried `pip install torch -f https://download.pytorch.org/who/torch_stable.html && pip install dist/*`?

Comment (Borda) [reply .github/workflows/install-pkg.yml]: seems did not help, so the issue may be elsewhere...

**Comment by Borda** [comment]
seems there is a random failure of `test_full_loop_ddp_spawn` even without touching it...
cc: @PyTorchLightning/core-contributors

**Pull request merged**